// Protocol buffer definitions used for testing grpcHoma.

syntax = "proto3";

package test;

service Test {
    rpc Sum(SumArgs) returns (SumResult) {}
    rpc SumMany(stream Value) returns (SumResult) {}
    rpc GetRange(Value) returns (stream Value) {}
    rpc IncMany(stream Value) returns (stream Value) {}
}

message SumArgs {
    int32 op1 = 1;
    int32 op2 = 2;
}

message SumResult {
    int32 sum = 1;
}

message Value {
    int32 value = 1;
}
